actor/src/main/scala/org/apache/pekko/dispatch/BatchingExecutor.scala

```diff
@@ -67,10 +67,10 @@ private[pekko] trait BatchingExecutor extends Executor {
   // invariant: if "_tasksLocal.get ne null" then we are inside Batch.run; if it is null, we are outside
   private[this] val _tasksLocal = new ThreadLocal[AbstractBatch]()
 
-  private[this] abstract class AbstractBatch extends ArrayDeque[Runnable](4) with Runnable {
+  private[this] abstract class AbstractBatch extends util.ArrayDeque[Runnable](4) with Runnable {
```

Comment:
(the reasoning here is to disambiguate between `scala.collection.mutable.ArrayDequeu` and `java.util.ArrayDeque` - that's fair I suppose. It might be even more readable to have the full `java.util.ArrayDeque` here instead of importing `java.util`)